Update `Project Loved` with removal process (#10209)
ppy/osu-wiki: wiki/Community/Project_Loved/en.md

Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category.

> Creators of said beatmaps are also able to request them to be removed from Loved category by reaching out to a Project Loved Team member or by posting in `#osu-loved` channel of the [osu! Discord server](https://discord.gg/ppy). The process of removal will take a minimum of 1 week and creator will receive a follow-up as it's finished.

Reviewer: ```suggestion
Creators of said beatmaps are also able to request them to be removed from the Loved category by reaching out to a [Project Loved Team](https://osu.ppy.sh/groups/31) member or by posting in `#osu-loved` channel of the [osu! Discord server](https://discord.gg/ppy). The removal process will take a minimum of 1 week and creator will receive a follow-up when it is finished.
```
